### PR TITLE
Feature/combine filter search

### PR DESCRIPTION
**Pull request by oscartison**
CLBV-80

### CODE REVIEW

**Comment by claire-lovisa** [comment]
@oscartison is it expected that this PR tries to merge to the master branch and not the development one ? Seems to bring a lot of changes with it, and some merge conflicts

**Comment by oscartison** [comment]
> @oscartison is it expected that this PR tries to merge to the master branch and not the development one ? Seems to bring a lot of changes with it, and some merge conflicts

My bad, should be fixed. @claire-lovisa

File: app/components/organization-status-select.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      (organizationStatus) => organizationStatus.id === id,
+    ).id;
+  }
+
+  @task
+  *loadOrganizationStatus() {
+    // Trick used to avoid infinite loop
+    // See https://github.com/NullVoxPopuli/ember-resources/issues/340 for more details
+
+    yield Promise.resolve();
```
Comment (claire-lovisa) [inline app/components/organization-status-select.js:40]: same, probably not useful

File: app/components/postal-select.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  findPostalByCode(code) {
+    return this.postalCodes.find(
+      (postalCode) => postalCode.postalCode === code,
+    );
+  }
+
+  @task
+  *loadPostalCodes() {
+    yield Promise.resolve();
```
Comment (claire-lovisa) [inline app/components/postal-select.js:36]: same, probably not useful

File: app/models/postal-code.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import Model, { attr } from '@ember-data/model';
+
+export default class IdentifierModel extends Model {
```
Comment (claire-lovisa) [inline app/models/postal-code.js:3]: `IdentifierModel` should be `PostalCodeModel` ?

**Review by claire-lovisa — APPROVED** [review]

**Pull request merged**